spec/models/question_spec.rb

```diff
@@ -1,5 +1,61 @@
 require 'rails_helper'
 
 RSpec.describe Question, type: :model do
+  subject { build(:question, user: user) }
+  let(:user) { build(:user) }
+  let(:question){create(:question, user: user)}
+
```

Comment:
:thought_balloon: you don't seem to have need for defining both `subject` and `question`